python/tchannel/messages/common.py

```diff
@@ -41,7 +41,7 @@
 class StreamState(IntEnum):
     init = 0x00,
     streaming = 0x01,
-    completed = 0x02
+    completed = 0x02,
```

Comment:
This isn't doing what you think it is :)

Within parenthesis, `('foo', 'bar')` is equivalent to `('foo', 'bar',)` because they represent the same tuple. BUT if you're not already in a tuple, then a dangling comma will implicitly create a single-element tuple. So what you have here means:

```
> StreamState.streaming == 0x01
False
> StreamState.streaming == (0x01,)
True
```